tests/integration/scripts/inotify_file.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/bash
+
+exit_err() {
+    echo -n "ERROR: "
+    echo "$@"
+    exit 1
+}
+
+# Install inotify-tools
+apt update -y || exit_err "Failed to update package list"
+apt install -y inotify-tools || exit_err "Failed to install inotify-tools"
```

Comment:
You might reduce the time of integration tests by installing requirements on the github workflow until we provide AMIs for it.

Reply:
good point! I'll do that, tks